widget/embedded/src/utils/meta.ts

```diff
@@ -17,6 +17,14 @@ export function getBlockchainShortNameFor(
     ?.shortName;
 }
 
+export function getBlockchainImage(
+  blockchainName: string,
+  blockchains: BlockchainMeta[]
+): string | undefined {
+  return blockchains.find((blockchain) => blockchain.name === blockchainName)
```

Comment:
iterating over all blockchains can be expensive specially when using it in the list of notifications which can contain several items. Can you please create a map of blockchains name and logo and use that instead.

Reply:
This type of performance improvements will be done in another task